### PR TITLE
Add openapi-python-client via grayskull

### PR DESCRIPTION
**Pull request by asford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/openapi-python-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - attrs >=21.0.0,<22.0.0
+    - autoflake >=1.4,<2.0
+    - black
+    - colorama >=0.4.3,<0.5.0
+    - httpx >=0.15.4,<0.19.0
+    - importlib_metadata >=2.0.0,<3.0.0
+    - isort >=5.0.5,<6.0.0
+    - jinja2 >=3.0.0,<4.0.0
+    - pydantic >=1.6.1,<2.0.0
+    - python
```
Comment (dopplershift) [inline recipes/openapi-python-client/meta.yaml:35]:
```suggestion
    - python >=3.6
```

**Comment by asford** [comment]
@dopplershift Thank you for the proactive review!

**Comment by asford** [comment]
It looks like the `importlib_metadata` requirement is version specific, which (I think) would force this module to not be noarch.
Is that correct?

Is the preferred solution for conda-forge to make this arch-specific?
In that case, I was seeing unexpected errors with the windows build.

Would it be acceptable to just pin to a higher python version?

**Comment by dopplershift** [comment]
Our usual solution is to just always require `importlib_metadata` to keep things noarch, even if it's not really needed on the later Python versions (we're only talking 10's of kb vs. 15 more package builds). I forget what Python version `importlib_metadata` becomes superfluous, but it's up to you as the recipe maintainer if you would prefer to go that route.

**Review by dopplershift — APPROVED** [review]
Looks good. Any reason this is still "WIP"?

**Comment by asford** [comment]
@dopplershift This should now be good to merge, thanks.

**Pull request merged**